### PR TITLE
add kubelet cadvisor container metrics 

### PR DESCRIPTION
**Pull request by szuecs**
to analyze container cpu throttling we need to scrape container metrics

### CODE REVIEW

File: cluster/senza-definition.yaml [hunk 1]
```diff
@@ -478,6 +478,17 @@ Resources:
         - Key: "kubernetes.io/cluster/{{ Arguments.ClusterID }}"
           Value: owned
     Type: AWS::EC2::SecurityGroupIngress
+  WorkerSecurityGroupIngressFromWorkerTocAdvisor:
```
Comment (linki) [inline cluster/senza-definition.yaml]: I think you could also add this to [this list](https://github.com/zalando-incubator/kubernetes-on-aws/blob/dev/cluster/senza-definition.yaml#L414).

**Comment by szuecs** [comment]
@linki done, thanks for the comment

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**